spel/scripts/amigen8-build.sh

```diff
@@ -9,7 +9,7 @@ PROGNAME="$(basename "$0")"
 AMIGENBOOTSIZE="${SPEL_AMIGENBOOTSIZE:-UNDEF}"
 AMIGENBRANCH="${SPEL_AMIGENBRANCH:-master}"
 AMIGENBUILDDEV="${SPEL_AMIGENBUILDDEV:-/dev/xvda}"
-AMIGENCHROOT="${SPEL_AMIGENCHROOT:-/mnt/ec2-root}"
+CHROOT="${SPEL_CHROOT:-/mnt/ec2-root}"
```

Comment:
Can we go the other way with this one, make this reference in amigen7 look more like amigen8? That will also require changing the env in the packer template from `SPEL_CHROOT` to `SPEL_AMIGENCHROOT`...

```suggestion
AMIGENCHROOT="${SPEL_AMIGENCHROOT:-/mnt/ec2-root}"
```

Reply:
Made the change to have us use amigenchroot and be more like amigen8